integration/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java

```diff
@@ -40,24 +41,24 @@
  * Notice that, all test begins with "IoTDB" is integration test. All test which will start the
  * IoTDB server should be defined as integration test.
  */
+@Category({LocalStandaloneTest.class})
```

Comment:
I wonder why do not add ClusterTest...

Reply:
> I wonder why do not add ClusterTest and RemoteTest...
> Good work! There are some class only marked by LocalStandaloneTest, are they could only run on local mode or we could add ClusterTest and RemoteTest also?

Yes, it can add. As the work of catching up with the master branch is continuous, I think all refactor work will be split into 2 temporary phases.  

Phase 1, 
- This PR should be reviewed.
- All tests should be annotated with the LocalStandaloneTest category.
- All these LocalStandaloneTest should be passed.
- No-brain tutorials should be done.
The milestone of phase 1 is this PR is merged. 

Phase 2, 
- All tests should be appropriately annotated with the ClusterTest category.
- All these ClusterTest should be passed.
- New IT should be followed by No-brain tutorials.
The milestone of phase 2 is all ITs are passed. 

I think phase 1 is done this week is appropriate. And phase 2 is not so urgent.

The above is already mentioned in the #integration-test channel of slack in Chinese. I will sync the progress in slack.

> I guess all test should annotated by three test modes? Unless the function we do not support in cluster.

Yes, not all tests can add ClusterTest and RemoteTest. 
Tests with local operations like restart or some descriptor's method invoking can't add for ClusterTest and RemoteTest and tests with config change can't add for RemoteTest.

> Some no brain tutorials are needed for others to work with the new IT framwork.

The no-brain tutorials are in progress. I will send the mail to [dev@iotdb.apache.org](mailto:dev@iotdb.apache.org) when finished. The mail will send tomorrow at the latest.

Reply:
Thanks!